spec/forms/two_factor_options_form_spec.rb

```diff
@@ -133,5 +133,28 @@
         end
       end
     end
+
+    context 'when a user signs up with phishing resistant requirement' do
+      let(:user) { build(:user) }
+      let(:enabled_mfa_methods_count) { 1 }
```

Comment:
I suspect it was copied from above `context` block, but I don't see that this variable has any relevance or impact on the specs in this block.

```suggestion
```

Reply:
I see that. Thanks!